docker-compose.yml

```diff
@@ -296,19 +296,17 @@ services:
       - COLLECTOR_OTLP_ENABLED=true
 
   keycloak:
-    image: quay.io/keycloak/keycloak:20.0.3
+    platform: ${KEYCLOAK_DOCKER_PLATFORM}
+    image: chorus-keycloak
     container_name: keycloak
-    hostname: keycloak
+    build:
+      context: ./keycloak
+      dockerfile: Dockerfile
+      args:
+        - keycloak_admin_password=${KEYCLOAK_ADMIN_PASSWORD}
```

Comment:
I am okay with hardcoding these thigns.  IN Chorus, the admin password will always be "password"...  and the admin username will laways either be "admin" or "admin@choruselectronics.com" ;-)    Fewer parameters and fewer configuration will help folks easier udnerstand what is happening...